packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -40,7 +40,10 @@
  * provides support for base view properties such as backgroundColor, opacity, etc.
  */
 public abstract class BaseViewManager<T extends View, C extends LayoutShadowNode>
-    extends ViewManager<T, C> implements BaseViewManagerInterface<T> {
+    extends ViewManager<T, C> implements BaseViewManagerInterface<T>, View.OnLayoutChangeListener {
+
+  private String transformOrigin;
+  private ReadableArray transform;
```

Comment:
Private variables use `m` as prefix
```suggestion
  private String mTransformOrigin;
  private ReadableArray mTransform;
```

Reply:
Oh, and these props should live on ReactViewGroup, not ViewManager. A ViewManager is responsible for many views.